AzureLoggingAlertsToTeamsWithStorage

```diff
@@ -0,0 +1,663 @@
+          "allowPermanentDelete": false,
+          "enabled": true,
+          "days": 7
+        },
+        "isVersioningEnabled": false,
+        "changeFeed": {
+          "enabled": false
+        },
+        "restorePolicy": {
+          "enabled": false
+        },
+        "containerDeleteRetentionPolicy": {
+          "enabled": true,
+          "days": 7
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Storage/storageAccounts/queueServices",
+      "apiVersion": "2021-09-01",
+      "name": "[concat(parameters('storageAccounts_azurealertsteamsmessage_name'), '/default')]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Storage/storageAccounts', parameters('storageAccounts_azurealertsteamsmessage_name'))]"
+      ],
+      "properties": {
+        "cors": {
+          "corsRules": []
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Storage/storageAccounts/tableServices",
+      "apiVersion": "2021-09-01",
+      "name": "[concat(parameters('storageAccounts_azurealertsteamsmessage_name'), '/default')]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Storage/storageAccounts', parameters('storageAccounts_azurealertsteamsmessage_name'))]"
+      ],
+      "properties": {
+        "cors": {
+          "corsRules": []
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Web/connections",
+      "apiVersion": "2016-06-01",
+      "name": "[parameters('connections_azuretables_name')]",
+      "location": "westeurope",
+      "dependsOn": [
+        "[resourceId('Microsoft.Storage/StorageAccounts', parameters('storageAccounts_azurealertsteamsmessage_name'))]"
+      ],
+      "kind": "V1",
+      "properties": {
+        "displayName": "[concat('https://', parameters('storageAccounts_azurealertsteamsmessage_name'), '.table.core.windows.net/HackathonTeamsMessage')]",
+        "statuses": [
+          {
+            "status": "Connected"
+          }
+        ],
+        "customParameterValues": {},
+        "nonSecretParameterValues": {
+          "storageaccount": "hackathonteamsmessage"
+        },
+        "createdTime": "2022-04-22T07:54:30.5039265Z",
+        "changedTime": "2022-04-22T07:54:30.6602556Z",
+        "api": {
+          "name": "[parameters('connections_azuretables_name')]",
+          "displayName": "Azure Table Storage",
+          "description": "Azure Table storage is a service that stores structured NoSQL data in the cloud, providing a key/attribute store with a schemaless design. Sign into your Storage account to create, update, and query tables and more.",
+          "iconUri": "[concat('https://connectoricons-prod.azureedge.net/releases/v1.0.1538/1.0.1538.2619/', parameters('connections_azuretables_name'), '/icon.png')]",
+          "brandColor": "#804998",
+          "id": "[concat('/subscriptions/06d56d01-0334-449a-8e17-1f46ade941d8/providers/Microsoft.Web/locations/westeurope/managedApis/', parameters('connections_azuretables_name'))]",
```

Comment:
Dit is nu ook fixed? (subscription id)

Reply:
Subscription id aangepast naar parameter